src/intro-to-curve-dodao-academy.yaml

```diff
@@ -96,7 +96,7 @@ steps:
             key: c3022c38
           - content: 'By creating tripool '
             key: 9f74db07
-        content: How does curve finance overcome the problem of slippage
+        content: How does curve finance overcome the problem of slippage that occurs due to volatality, while order being executed ?
```

Comment:
Slippage is not related to gauge weights. Gauge weights are for rewards

Reply:
yes sir, here the answer is 2nd option.